doc/ale.txt

```diff
@@ -1938,6 +1938,22 @@ g:ale_root                                                         *g:ale_root*
   LSP linter, it will not run.
 
 
+g:ale_send_to_neovim_diagnostics             *g:ale_send_to_neovim_diagnostics*
+
+  Type: Number
+  Default: 0
+
+  If enabled, this option will disable ALE's standard UI, and instead send
+  all linter output to Neovim's diagnostics API. This allows you to collect
+  errors from nvim-lsp, ALE, and anything else that uses diagnostics all in
+  one place.
```

Comment:
We need to list out all of the options here that will be disabled by virtue of turning this one one, with links to the documentation for each. That way people will know exactly what this does.